.bazelrc

```diff
@@ -54,6 +54,9 @@ build:sanitizer --copt -O1
 build:sanitizer --//bazel:has_sanitizers=True
 # seastar has to be run with system allocator when using sanitizers
 build:sanitizer --@seastar//:system_allocator=True
+# This is currently required to get ASAN to work, but ideally in the future we can get
+# the sanitizers working in release mode
```

Comment:
i think this would be the `dev` mode we've talked about? i was under the impression that sanitziers don't work in -O2, but maybe i'm mixing some things up.

Reply:
Yeah also the proposed CI mode (which is maybe the dev mode?) that @travisdowns mentioned.

Why wouldn't they work in O2? If they don't that's fine, but I was seeing linker failures with -c opt so I think it's more of a static vs dynamic linking issue

Reply:
yeh im not saying they don't work, i just have a memory about them not being as effective. but i may be hallucinating